syne_tune/optimizer/baselines.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict
+
+from syne_tune.optimizer.schedulers.fifo import FIFOScheduler
+from syne_tune.optimizer.schedulers.hyperband import HyperbandScheduler
+from syne_tune.optimizer.schedulers.multiobjective.moasha import MOASHA
+from syne_tune.optimizer.schedulers.pbt import PopulationBasedTraining
+
```

Comment:
Do we also want to have a baseline for sync Hyperband? I could write it afterwards.

Reply:
Good question. I would think that we do not because the method is strongly outperformed by async counterpart. 

I guess people will use it when they want to compare against async methods, in this sense those are probably researchers experts who know what they are doing and can configure Fifo/HB schedulers to be whatever they want (as opposed to users who just want ASHA to tune their algo).